common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -151,12 +151,28 @@ def has(self, key):
         else:
             return False
 
+    def __repr__(self):
+        return "MongoKeyValueStore{!r}<{!r}, {!r}>".format(
+            (self._data, self._parent, self._children, self._metadata),
+            self._fields,
+            self.inherited_settings
+        )
+
```

Comment:
`__repr__` should be whatever would be useful to developers.  There's no need for them to be distinct for distinct objects, but if you find it helpful to see the identity, then you can include that in the string.  Some people (Cale among them) try to make the `__repr__` output look like a Python expression to create the object.  I prefer a "<ClassName value value value>" syntax, since we don't actually use `__repr__` output to recreate the objects.  But either way, the string is enough to tell the developer what type of thing it is, and what its important distinguishing values are.